snoop/data/analyzers/archives.py

```diff
@@ -7,16 +7,43 @@
 
 
 SEVENZIP_KNOWN_TYPES = {
-    'application/zip',
-    'application/rar',
+    # these are known to work
     'application/x-7z-compressed',
+    'application/zip',
     'application/x-zip',
+    'application/x-rar',
+    # these might not work properly
     'application/x-gzip',
-    'application/x-zip-compressed',
-    'application/x-rar-compressed',
+    'application/x-bzip2',
+    'application/x-tar',
+}
+
+READPST_KNOWN_TYPES = {
+    'application/x-hoover-pst',
 }
 
 
+def is_archive(mime_type):
+    return mime_type in SEVENZIP_KNOWN_TYPES + READPST_KNOWN_TYPES
```

Comment:
For the sake of optimization, let's concatenate them once, and save the result to a global.